src/js/tech/html5.js

```diff
@@ -573,6 +577,33 @@ class Html5 extends Tech {
   }
 
   /**
+   * Proxy iOS `webkitbeginfullscreen` and `webkitendfullscreen` into
+   * `fullscreenchange` event
+   *
+   * @private
+   * @method proxyWebkitFullscreen_
+   */
+  proxyWebkitFullscreen_() {
+    if ('webkitDisplayingFullscreen' in this.el_) {
```

Comment:
I would do `if (!'webkitDisplayingFullscreen' in this.el_)` if that works and return at the top